app/app/src/main/java/tech/ziasvannes/safechat/presentation/components/ContactListItem.kt

```
@@ -62,18 +66,30 @@ fun ContactListItem(
-                        imageVector = Icons.Default.Person,
-                        contentDescription = "Avatar for ${contact.name}",
-                        tint = MaterialTheme.colorScheme.onSurfaceVariant,
-                        modifier = Modifier.size(24.dp)
-                )
+            if (contact.avatar != null) {
+                val imageBitmap =
+                        try {
+                            val decodedBytes = Base64.decode(contact.avatar, Base64.DEFAULT)
+                            BitmapFactory.decodeByteArray(decodedBytes, 0, decodedBytes.size)
+                                    ?.asImageBitmap()
+                        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)